app/models/proposal.rb

```diff
@@ -80,7 +97,11 @@ def users
   # returns the Approval
   def add_approver(email)
     user = User.for_email(email)
-    self.approvals.create!(user_id: user.id)
+    if self.parallel? || self.approvals.empty?
+      self.approvals.create!(user_id: user.id, status: 'actionable')
+    else
+      self.approvals.create!(user_id: user.id)
+    end
```

Comment:
I don't think we want to do this distinction inside of this method...`actionable` should be set on the first Approval for a Proposal based on the `position`, not the order in which the Approval is added.